src/legacy/core_plugins/timelion/index.js

```diff
@@ -46,63 +53,107 @@ export default function (kibana) {
+          description: i18n.translate('timelion.uiSettings.defaultIndexDescription', {
+            defaultMessage: 'Default elasticsearch index to search with {esParam}',
+            values: { esParam: '.es()' }
+          }),
           category: ['timelion'],
         },
         'timelion:target_buckets': {
-          name: 'Target buckets',
+          name: i18n.translate('timelion.uiSettings.targetBucketsLabel', {
+            defaultMessage: 'Target buckets'
+          }),
           value: 200,
-          description: `The number of buckets to shoot for when using auto intervals`,
+          description: i18n.translate('timelion.uiSettings.targetBucketsDescription', {
+            defaultMessage: 'The number of buckets to shoot for when using auto intervals'
+          }),
           category: ['timelion'],
         },
         'timelion:max_buckets': {
-          name: 'Maximum buckets',
+          name: i18n.translate('timelion.uiSettings.maximumBucketsLabel', {
+            defaultMessage: 'Maximum buckets'
+          }),
           value: 2000,
-          description: `The maximum number of buckets a single datasource can return`,
+          description: i18n.translate('timelion.uiSettings.maximumBucketsDescription', {
+            defaultMessage: 'The maximum number of buckets a single datasource can return'
+          }),
           category: ['timelion'],
         },
         'timelion:default_columns': {
-          name: 'Default columns',
+          name: i18n.translate('timelion.uiSettings.defaultColumnsLabel', {
+            defaultMessage: 'Default columns'
+          }),
           value: 2,
-          description: `Number of columns on a timelion sheet by default`,
+          description: i18n.translate('timelion.uiSettings.defaultColumnsDescription', {
+            defaultMessage: 'Number of columns on a timelion sheet by default'
+          }),
           category: ['timelion'],
         },
         'timelion:default_rows': {
-          name: 'Default rows',
+          name: i18n.translate('timelion.uiSettings.defaultRowsLabel', {
+            defaultMessage: 'Default rows'
+          }),
           value: 2,
-          description: `Number of rows on a timelion sheet by default`,
+          description: i18n.translate('timelion.uiSettings.defaultRowsDescription', {
+            defaultMessage: 'Number of rows on a timelion sheet by default'
+          }),
           category: ['timelion'],
         },
         'timelion:min_interval': {
-          name: 'Minimum interval',
+          name: i18n.translate('timelion.uiSettings.minimumIntervalLabel', {
+            defaultMessage: 'Minimum interval'
+          }),
           value: '1ms',
-          description: `The smallest interval that will be calculated when using "auto"`,
+          description: i18n.translate('timelion.uiSettings.minimumIntervalDescription', {
+            defaultMessage: 'The smallest interval that will be calculated when using "auto"'
```

Comment:
@timroes should we put "auto" to values?

Reply:
I would say, just add a description `"auto" is a technical value in that context, that should not be translated.`.